LICENSE.md

```diff
@@ -0,0 +1,31 @@
+BSD 3-Clause License
+
+Copyright (c) 2014-2020 Jolla Ltd.
+Copyright (c) 2025 Jolla Mobile Ltd.
+All rights reserved.
```

Comment:
I'm not sure if "all rights reserved" is appropriate here, as it seems slightly contradictory with the license.

Reply:
Removed the line.